platform/android/src/geojson/polygon.cpp

```diff
@@ -12,19 +12,19 @@ mapbox::geojson::polygon Polygon::convert(jni::JNIEnv &env, jni::Object<Polygon>
     mapbox::geojson::polygon polygon;
 
     if (jPolygon) {
-        auto jPositionListsList = Polygon::getCoordinates(env, jPolygon);
+        auto jPositionListsList = Polygon::coordinates(env, jPolygon);
         polygon = Polygon::convert(env, jPositionListsList);
         jni::DeleteLocalRef(env, jPositionListsList);
     }
 
     return polygon;
 }
 
-mapbox::geojson::polygon Polygon::convert(jni::JNIEnv &env, jni::Object<java::util::List/*<java::util::List<Position>>*/> jPositionListsList) {
+mapbox::geojson::polygon Polygon::convert(jni::JNIEnv &env, jni::Object<java::util::List/*<java::util::List<Point>>*/> jPointListsList) {
     mapbox::geojson::polygon polygon;
 
-    if (jPositionListsList) {
-        auto multiLine = MultiLineString::convert(env, jPositionListsList);
+    if (jPointListsList) {
+        auto multiLine = MultiLineString::convert(env, jPointListsList);
```

Comment:
@tobrun  could it be that DeleteLocalRef() for multiline was and is missing here between lines 31 & 32

Reply:
yes it seems that we aren't DeleteLocalRef on `jPointListsList ` as we are doing in the other classes eg. https://github.com/mapbox/mapbox-gl-native/blob/0ee6c2c5c0fc4303ff3128890eef1ad3db471deb/platform/android/src/geojson/multi_point.cpp#L17. Could you add that?

Reply:
@tobrun Looks like a false alarm on my side:
``` 
/src/platform/android/src/geojson/polygon.cpp:32:34: error: no viable conversion from 'mapbox::geometry::multi_line_string<double, std::vector>' to 'jni::jobject *'
          jni::DeleteLocalRef(env, multiLine);
                                   ^~~~~~~~~```